src/include/platform/CHIPDeviceEvent.h

```diff
@@ -395,6 +397,7 @@ typedef void (*AsyncWorkFunct)(intptr_t arg);
 #include <system/SystemEvent.h>
 #include <system/SystemLayer.h>
 #include <system/SystemPacketBuffer.h>
+#include <wifipaf/WiFiPAFRole.h>
```

Comment:
Shouldn't this include header also be enclosed in a define? Since this include is not needed for non CHIP_DEVICE_CONFIG_ENABLE_WIFIPAF

Reply:
I will add it